.github/github_org_control/requirements.txt

```diff
@@ -1,2 +1,3 @@
-PyGithub==1.55
-ldap3==2.7
+-c ../../src/bindings/python/constraints.txt
```

Comment:
do you really think we need to have repository-level constraints? 
Some files like `.github/github_org_control/requirements.txt` are used in completely different environment rather than product and I don't think we need to put everything under a single umbrella

Reply:
That's a good point, I guess I got lost in the dream of having everything in one place. I removed the requirement files which do not belong to this PR.